src/components/Calculator.js

```diff
@@ -9,35 +10,70 @@ const Calculator = () => {
     next: null,
     operation: null,
   });
+  const [quotes, setQuotes] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const options = {
+          method: 'GET',
+          headers: { 'x-api-key': 'btAEiRrit9u1L1sZDM0Oew==si2rJMo7DAvCioHN' },
+        };
+        const res = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=happiness',
+          options,
+        );
+        const json = await res.json();
+        setQuotes(json);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setQuotes, setIsLoading]);
+
+  if (hasError) return <div>Something went wrong!</div>;
+
+  if (isLoading) return <div>Loading...</div>;
+
   return (
     <div className="container">
-      <Display currentValue={currentValue} />
-      <InputValues
-        data={data}
-        setData={setData}
-        setCurrentValue={setCurrentValue}
-        values={[
-          'AC',
-          '+/-',
-          '%',
-          '÷',
-          '7',
-          '8',
-          '9',
-          'x',
-          '4',
-          '5',
-          '6',
-          '-',
-          '1',
-          '2',
-          '3',
-          '+',
-          '0',
-          '.',
-          '=',
-        ]}
-      />
+      <div className="quote">
+        <Quotes quotes={quotes} />
```

Comment:
- [ ] Kindly import the new component (Quotes.js) into the main component (App.js) next to the Calculator component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Import%20the%20new%20component%20into%20your%20main%20component%20(next%20to%20the%20Calculator)).